src/views/Contact.jsx

```diff
@@ -23,7 +23,7 @@ const ContactForm = styled(Paper)(({ theme }) => ({
   padding: theme.spacing(4),
   maxWidth: "600px",
   margin: "auto",
-  backgroundColor: "linear-gradient(135deg, #1F1C2C 10%, #232526 100%)",
+  backgroundColor: "rgba(20, 20, 20, 0.8)",
```

Comment:
here

Reply:
I'm not sure what that looked like.  since what I got was the screenshot in #318 which had no gradient

Reply:
@mwango-phoenix, if you want to change the design further, you can. but the present is not looking good.